concourse/pipelines/create-deployer.yml

```diff
@@ -413,6 +413,7 @@ jobs:
     serial: true
     plan:
     - get: paas-cf
+      passed: [vpc]
```

Comment:
Since this job actually runs both the vpc and concourse terraform runs, it should probably pin to concourse-terraform.